mirrors_countme/constants.py

```diff
@@ -20,7 +20,6 @@
 
 # Feb 11 2020 was the date that we branched F32 from Rawhide, so we've decided
 # to use that as the starting week for countme data.
-COUNTME_START_TIME = 1581292800  # =Mon Feb 10 00:00:00 2020 (UTC)
```

Comment:
This bit, is probably worth keeping. Maybe as just a comment if we really want to delete it, but leaving the constant around is probably fine too.